### DISCUSSION TITLE
Extracting parameters of Repositories for research purpose

### DISCUSSION THREAD

**Discussion by senmalkis** [community/community#24886] [Apps, API and Webhooks]
<p>Dear All,</p>
<p>I am working on a research to study the success of repo/projects using various parameters including number of branches, issues, releases, collaborators, contributors, followers, following etc. Any one earlier did any query or releated request, please advise me. Thanks in advance. </p>
<p>Regards,</p>
<p>Senthil</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/senmalkis">@senmalkis</a>,</p>
<p>Thanks for being here! Here is the link to the <a href="https://developer.github.com/v3/#schema" rel="nofollow noopener">API’s documentation</a> concerning params. I hope this helps! </p>
<p>Best,</p>
<p>Andrea</p>

**Comment by senmalkis** [comment]
<p><span class="mention">@Andrea</span>, </p>
<p>Thank you very much for a quick reply. </p>
<p>Regards,</p>
<p>Senthil</p>
